spec.html

```diff
@@ -2365,7 +2365,7 @@ <h1>
           <emu-alg>
             1. If _exponent_ &lt; *0*<sub>ℤ</sub>, throw a *RangeError* exception.
             1. If _base_ is *0*<sub>ℤ</sub> and _exponent_ is *0*<sub>ℤ</sub>, return *1*<sub>ℤ</sub>.
-            1. Return the BigInt value that represents ℝ(_base_) raised to the power ℝ(_exponent_).
+            1. Return the BigInt value for ℝ(_base_) raised to the power ℝ(_exponent_).
```

Comment:
This will help with association of "the BigInt value for":

```suggestion
            1. Return ℤ(ℝ(_base_) raised to the power ℝ(_exponent_)).
```

Alternatively, we could make it two steps.

Reply:
> This will help with association of "the BigInt value for":
> 
> 
> 
> ```suggestion
> 
>             1. Return ℤ(ℝ(_base_) raised to the power ℝ(_exponent_)).
> 
> ```
> 
> 
> 
> Alternatively, we could make it two steps.

Reply:
Help me